README.md

```diff
@@ -142,6 +142,23 @@ steps:
       MERGE_METHOD: MERGE
 ```
 
+### Specify the semantic versioning category to use
+
+By default, this GitHub Action assumes merge category is `MAJOR`. This feature
+looks at the title of the PR to figure out what is being updated. We expect a
+dependabot style title such as: `bump @types/jest from 26.0.12 to 27.0.13`. You
```

Comment:
Well, no one expects such PRs, this Action is not only for Dependabot, up until now I don't think it favorized it in any way. Are we going to support Renovate, Gardener and other tools as well? Would you consider changing renaming `MERGE_CATEGORY` to `PRESET` and values to `DEPENDABOT_MAJOR`, `DEPENDABOT_MINOR`, ...? That way it would be communicated better that it's open to adding more presets, e.g. presets for languages that don't use semantic versioning at all.

Reply:
Initially I'm not planning on supporting Renovate, Gardener or others. If such a request would come in, then we can extend this logic to support those platforms as well.

Reply:
I renamed `MERGE_CATEGORY` to `PRESET`

Reply:
@merlinnot maybe we could use something like `SEMANTIC_MAJOR` instead of `DEPENDABOT_MAJOR` WDYT?

Since some (not all) of the other tools will be using semantic versioning

Reply:
Well, I just want to avoid any breaking changes in the future. It seems to me that the feature we built here is very specific to Dependabot, especially that it relies on a very specific PR title that Dependabot creates. By naming it `DEPENDABOT_MAJOR` we can manage expectations a little better. If we call it `SEMANTIC_MINOR`, one can also expect that it will work with Renovate (merge only patch and minor from it).

Reply:
Gotcha. I'll change it 👍

Reply:
Done!